src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -259,7 +259,7 @@ public function getEnv(string $prefix, string $name, \Closure $getEnv)
                 throw new RuntimeException(sprintf('Invalid URL in env var "%s".', $name));
             }
             if (!isset($params['scheme'], $params['host'])) {
-                throw new RuntimeException(sprintf('Invalid URL env var "%s": schema and host expected, "%s" given.', $name, $env));
+                throw new RuntimeException(sprintf('Invalid URL in env var "%s": scheme and host expected.', $name, $env));
```

Comment:
Is it intended that there is no placeholder for `$env` anymore?

Reply:
yes: it could leak sensitive details and is not consistent with the previous error message anyway